.github/workflows/build.yml

```diff
@@ -17,7 +17,7 @@ name: Build
 
 on:
   push:
-    branches: [ main, develop ]
+    branches: [ main, develop, master ]
```

Comment:
I thought to replace the `main` and `develop` as they don't exist. But doesn't matter that much 😉